tests/conftest.py

```diff
@@ -35,8 +35,8 @@
 def _download_testdata():
     fhtar = tempfile.NamedTemporaryFile(delete=False, suffix=".tar.gz")
 
-    r = requests.get(TESTDATA_URL)
-    fhtar.write(r.content)
+    req = requests.get(TESTDATA_URL)
```

Comment:
Don't quite remember why we changed these bits.

Reply:
I think this might be a flake8 issue :) Happy for you to leave this in